src/engine/identification.md

```diff
@@ -0,0 +1,73 @@
+# Engine API -- Client Identification
+
+Engine API structures and methods specified for client identification.
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Client Identification](#client-identification)
+- [Structures](#structures)
+  - [ClientCode](#clientcode)
+  - [ClientIdentificationV1](#clientidentificationv1)
+- [Methods](#methods)
+  - [engine_clientIdentificationV1](#engine_clientidentificationv1)
+    - [Request](#request)
+    - [Response](#response)
+    - [Specification](#specification)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Client Identification
+
+To facilitate a more accurate measurement of execution layer client diversity statistics, execution clients **SHOULD** support the methods described in this document.
+
+## Structures
+
+### ClientCode
+
+This enum defines a standard for specifying a client with just two letters. Clients teams which have a code reserved in this list **MUST** use this code when identifying themselves. The code is specified here only to facilitate standardization and NOT to imply that these are the only supported Ethereum clients. Any clients not listed here are free to use any two letters which don't collide with an existing client code. They are encouraged to make a PR to this repo to reserve their own code. Existing codes are as follows:
+
+ - `BU`: besu
+ - `EJ`: ethereumJS
+ - `EG`: erigon
+ - `GE`: go-ethereum
```

Comment:
May as well reserve `GR` here for Grandine, as it's a client that exists regardless of its open-source status

Reply:
done! :)